src/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import TeamMembers from './TeamMembers'
+import Form from './Form'
+
+const initialFormValues = {
+  name: '',
+  email: '',
+  role: '',
+}
+
+export default function App() {
+  const [teamMembersList, setTeamMembersList] = useState([]);
+
+  const [formValues, setFormValues] = useState(initialFormValues);
+
+  const updateForm = (inputName, inputValue) => {
+    setFormValues({...formValues, [inputName]: inputValue})
+  }
+
+  const submitForm = () => {
+    const newMember = {
+      name: formValues.name.trim(),
+      email: formValues.email.trim(),
+      role: formValues.role.trim(),
```

Comment:
You wouldn't want to use a .trim() on the role since it is not an input! It is a fixed choice so no need for it :)